assets/base/i18n/en.json

```diff
@@ -360,6 +360,11 @@
   "mobile.notification_settings.ooo_auto_responder": "Automatic Direct Message Replies",
   "mobile.notification_settings.save_failed_description": "The notification settings failed to save due to a connection issue, please try again.",
   "mobile.notification_settings.save_failed_title": "Connection issue",
+  "mobile.oauth.failed_to_login": "Failed to login",
+  "mobile.oauth.failed_to_open_link": "Failed to open the link",
+  "mobile.oauth.restart_login": "Restart login",
```

Comment:
@anurag6713 Can you confirm how this one is being used in the UI or what would cause this message?